docs/content/en/docs/concepts/support-versions.md

```diff
@@ -112,12 +112,15 @@ However, if you create your management cluster with version v0.15.0 and then upg
 
 No. We perform regular upgrade reliability testing for sequential version upgrade (e.g. going from version 0.9 to 0.10, then from version 0.10 to 0.11), but we do not perform testing on non-sequential upgrade path (e.g. going from version 0.9 directly to 0.11). You should _not_ skip minor versions during cluster upgrade. However, you can choose to skip patch versions.
 
-### What kind of fixes are back-ported to the previous versions?*
+### What kind of fixes are back-ported to the previous versions?
 Back-ported fixes include CVE patches and bug fixes for the Amazon EKS Anywhere components and the Kubernetes versions that are supported by the corresponding Amazon EKS Anywhere versions. 
 
 ### What happens on the end of support date for a Kubernetes version?
-On the end of support date, you can still create a new cluster with the unsupported Kubernetes version using an old version of the Amazon EKS Anywhere toolkit that was released with this Kubernetes version. Any existing Amazon EKS Anywhere clusters with the unsupported Kubernetes version will continue to function. If you have the [Amazon EKS Anywhere Enterprise subscription](https://aws.amazon.com/eks/eks-anywhere/pricing/), AWS Support will continue to provide troubleshooting support and configuration guidance to those clusters as long as their Amazon EKS Anywhere versions are still being supported. However, you will not be able to receive CVE patches or bug fixes for the unsupported Kubernetes version.
+On the end of support date, you can still create a new cluster with the unsupported Kubernetes version using an old version of the Amazon EKS Anywhere toolkit that was released with this Kubernetes version. Any existing Amazon EKS Anywhere clusters with the unsupported Kubernetes version will continue to function. However, you will not be able to receive CVE patches or bug fixes for the unsupported Kubernetes version.
+
+### What EKS Anywhere versions are supported if you have the Amazon EKS Anywhere Enterprise Subscription?
+
+If you have the [Amazon EKS Anywhere Enterprise Subscription](https://aws.amazon.com/eks/eks-anywhere/pricing/), AWS Support will provide troubleshooting support and configuration guidance for your licensed cluster, irrespective of the specific EKS Anywhere version it's running on. However, as the CVE patches and bug fixes are only back-ported to the latest EKS Anywhere version (version n), it is highly recommended to keep your cluster updated with the latest EKS Anywhere release. With Enterprise Subscription, AWS Support can assist you in upgrading your licensed cluster to the most recent EKS Anywhere version.
 
```

Comment:
Can we add the following FAQ

### Does EKS extended support for Kubernetes versions apply to EKS Anywhere clusters?

No. EKS extended support for Kubernetes versions does not apply to EKS Anywhere at this time. To request this capability, please comment or upvote on this [EKS Anywhere GitHub issue](https://github.com/aws/eks-anywhere/issues/6793).